src/slowmokit/methods/neighbors/bernoulli_nb/bernoulli_nb.cpp

```diff
@@ -0,0 +1,64 @@
+/**
+ * @file methods/neighbors/bernoulli_nb/nernoulli_nb.cpp
+ *
+ * Implementation of the Bernoulli Naive Bayes main program
+ */
+#include "bernoulli_nb.hpp"
+
+template<class T>
+    double prior_prob(std::vector<T> y_train,int label){   // Prior-Probability P(y)
+        int sum=0;
+        for(int i=0;i<y_train.size();i++){
+            if(y_train[i]==label){
+                sum += 1;
+            }
+        }
+        return sum/double(y_train.size());
+    }
+
+template<class T>
+    double conditional_prob(std::vector<std::vector<T>> x_train,std::vector<T> y_train,int feature_col,int feature_val,int label){ // Conditional Probability
+    // P(x=f1 / y=class) 
+        int denominator=0,numerator=0;
+        for(int i=0;i<y_train.size();i++){
+            if(y_train[i]==label){
+                denominator++;
+                if(x_train[i][feature_col]==feature_val){
+                    numerator++;
+                }
+            }
+        }
+        return numerator/double(denominator);
+    }
+
+template<class T>
+    int predict(std::vector<std::vector<T>> x_train,std::vector<T> y_train,std::vector<T> x_test){
+        int n_features = x_train[0].size();
```

Comment:
According to our conventions, variable and function names should be in camel case so make these changes wherever required